Sources/SwiftSyntaxBuilder/generated/BuildableNodes.swift

```diff
@@ -381,7 +381,9 @@ extension DoStmtSyntax {
       leadingTrivia: Trivia? = nil, 
       unexpectedBeforeDoKeyword: UnexpectedNodesSyntax? = nil, 
       doKeyword: TokenSyntax = .keyword(.do), 
-      unexpectedBetweenDoKeywordAndBody: UnexpectedNodesSyntax? = nil, 
+      unexpectedBetweenDoKeywordAndThrowsClause: UnexpectedNodesSyntax? = nil, 
+      throwsClause: ThrowsClauseSyntax? = nil, 
+      unexpectedBetweenThrowsClauseAndBody: UnexpectedNodesSyntax? = nil, 
```

Comment:
Also related to https://github.com/apple/swift-syntax/pull/2379#discussion_r1414399964, this needs a compatibility layer or a mention in the release notes.